test-assets/test-projects/MSTestProject/TestProgram.cs

```diff
@@ -12,7 +12,7 @@ public class MainTest
         public void Test()
         {
             Assert.IsTrue(true);
-        }
+        } 
```

Comment:
This space (and the ones in the other test projects) are not a typo. They're there for testing the boundary cases.